sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -432,16 +431,18 @@ private static float setDefaultConfidenceValue(Float confidence) {
     private static Map<String, FormField> toFieldValueObject(Map<String, FieldValue> valueObject,
                                                              List<ReadResult> readResults) {
         Map<String, FormField> fieldValueObjectMap = new TreeMap<>();
-        valueObject.forEach((key, fieldValue) ->
-            fieldValueObjectMap.put(key,
-                setFormField(key,
-                    new FieldData(fieldValue.getText(),
-                        toBoundingBox(fieldValue.getBoundingBox()),
-                        fieldValue.getPage(),
-                        setReferenceElements(fieldValue.getElements(), readResults)),
-                    fieldValue,
-                    readResults)
-            ));
+        valueObject.forEach((key, fieldValue) -> {
```

Comment:
In theory this could happen to any type and not only object right?

Reply:
yes and that check is already present [here](https://github.com/Azure/azure-sdk-for-java/blob/hotfix/azure-ai-formrecognizer_3.1.1/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java#L313).

The check added now, is for the elements encountered in looping over the MAP object.

Reply:
I think array is ok? but I see a similar thing where text is also considered: https://github.com/Azure/azure-sdk-for-java/blob/3da88172dcc4dc4317385299958e1d248d13ca82/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java#L465

Reply:
updated